### PR TITLE
[DEV-2.3] Word Verification Done

### PR DESCRIPTION
**Pull request by evanMWJeong**
User Story 2.3 Verifying Word

### CODE REVIEW

**Review by gideonmehna — COMMENTED** [review]
The Code looks good to me. I wish there was a way we could load this from a separate class but I think we could add this to our product backlog instead.

File: views/BoggleView.java [hunk 1]
```diff
@@ -405,47 +405,47 @@ private void launchControls(){
         enterButton.setOnAction(e -> {
             if (inputWord[0].isEmpty()) inputWord[0] = textInputField.getText();
             wordInput.setText(inputWord[0]);
-//            HBox box1 = new HBox(5);
```
Comment (imranmuyingo) [inline views/BoggleView.java:408]: This is redundant so it can be deleted

**Review by imranmuyingo — CHANGES_REQUESTED** [review]
This is good work! The only thing left to complete would be to delete redundant code.

**Review by imranmuyingo — COMMENTED** [review]
This is good work. All that is left to do is to delete redundant code.

**Comment by imranmuyingo** [comment]
The branch also has merge conflicts that need to be fixed

**Review by imranmuyingo — APPROVED** [review]

**Pull request merged**